pkg/detectors/skilljar/skilljar_test.go

```diff
@@ -0,0 +1,161 @@
+//go:build detectors
+// +build detectors
+
+package skilljar
+
+import (
+	"context"
+	"fmt"
+	"github.com/google/go-cmp/cmp"
+	"github.com/google/go-cmp/cmp/cmpopts"
+	"testing"
+	"time"
+
+	"github.com/trufflesecurity/trufflehog/v3/pkg/detectors"
+
+	"github.com/trufflesecurity/trufflehog/v3/pkg/common"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/pb/detectorspb"
+)
+
+func TestSkilljar_FromChunk(t *testing.T) {
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second*5)
+	defer cancel()
+	testSecrets, err := common.GetSecret(ctx, "trufflehog-testing", "detectors4")
```

Comment:
@chuglo would you mind changing this to `detectors5`? 
```suggestion
	testSecrets, err := common.GetSecret(ctx, "trufflehog-testing", "detectors5")
```